rise/static/main.js

```diff
@@ -663,32 +705,18 @@ define([
     var help_button = $('<i/>')
         .attr('id','help_b')
         .attr('title','Reveal Shortcuts Help')
-        .addClass('fa-question fa-4x fa')
+        .addClass('fa-question fa-3x fa')
```

Comment:
Why are you changing the size here? Smaller things fit better with the chalkboard stuff?

Reply:
IIRC, yes that's the spirit, there are new buttons related to chalkboard, so this looked better, but it's clearly a matter of taste, feel free to undo of course